sh_scrapy/commands/shub_image_info.py

```diff
@@ -35,8 +35,23 @@ def add_options(self, parser):
     def run(self, args, opts):
         result = {
             'project_type': 'scrapy',
-            'spiders': sorted(self.crawler_process.spider_loader.list())
+            'spiders': sorted(self.crawler_process.spider_loader.list()),
         }
+        try:
+            from scrapy_spider_metadata import get_spider_metadata
+        except ImportError:
+            pass
```

Comment:
It seems we lack tests on CI which check a case where the library is not installed

Reply:
3.6 and 3.7 :)

Reply:
Why isn't this line detected as covered?

Reply:
None of run() is detected as covered: https://app.codecov.io/github/scrapinghub/scrapinghub-entrypoint-scrapy/commit/ebf23f19ff3bc1853a35f001a8b2df7e1b1ec63b/blob/sh_scrapy/commands/shub_image_info.py

Not sure if codecov can detect coverage for code run as a separate process.